contracts/contracts/Utilities.sol

```diff
@@ -0,0 +1,37 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity >=0.8.0 <0.9.0;
+pragma abicoder v2;
+
+contract Utilities {
```

Comment:
No a huge fan of this name, very generic. Maybe `Functions`, `FunctionsUtilities` instead?

We could also add this to an existing contract such as `VerificationGateway` or `BLSExpander` so we can avoid another contract deploy.

Reply:
Hmm. They're very generic utilities though. In fact I'm kinda hoping this already exists in some form and we can use that instead of this.

Anyway, I renamed it to `AggregatorUtilities`. Wdyt?

Reply:
> We could also add this to an existing contract such as VerificationGateway or BLSExpander so we can avoid another contract deploy.

Adding to `VerificationGateway` would increase the audit burden (and complexity that may be considered by users) though.

For `BLSExpander`, it really doesn't fit. That contract is about compressing calldata for `processBundle`.

Also, this may be out of date, but I believe there was some concern about the maximum allowed size of a contract and pushing up against that limit.

Is there much benefit in avoiding a contract deploy? Is it more efficient to deploy A+B functions into one contract than A functions into contract A and B functions into contract B?